src/components/app/routes/data/utils.js

```diff
@@ -310,9 +311,8 @@ export async function ensureActiveEnterpriseCustomerUser({
     try {
       await updateUserActiveEnterprise({ enterpriseCustomer: nextActiveEnterpriseCustomer });
     } catch (error) {
-      logError(`Unable to update active enterprise customer: ${nextActiveEnterpriseCustomer}
-      for user ${authenticatedUser.userId}
-      ${error.message}`);
+      error.message = `Unable to update active enterprise customer: ${nextActiveEnterpriseCustomer} for user ${authenticatedUser.userId}: ${error.message}`;
+      logError(error);
```

Comment:
[pondering whether this is the pattern we should be using `logError` with (or similar) to ensure we get the original `error.stack` included in Datadog]